### PR TITLE
Exponential Gradient strategy + tests

### LINKED ISSUE #7: Exponential Gradient
Implementation and testing of the Exponential Gradient strategy.

- [ ] Implement strategy
- [ ] Write tests for strategy

Once you complete the implementation + tests in a branch, go ahead and link a PR to this issue to close this issue automatically when the PR is merged.

### PR DESCRIPTION
**Pull request by SayyantRath**

### CODE REVIEW

File: olps/strategies/exp_grad_strategy.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from .strategy import Strategy
+from ..datasources.datasource import DataSource
+import numpy as np
+
+class ExpGradStrategy(Strategy):
+
+    learning_rate = 0.1
+    # TODO implement
```
Comment (adityarathod) [inline olps/strategies/exp_grad_strategy.py]: you don't need this lmao

**Review by adityarathod — APPROVED** [review]
LGTM!
